protractor_demo/pages/translate.ts

```diff
@@ -1,39 +1,54 @@
-import { browser, ExpectedConditions, protractor } from 'protractor';
+import { $, browser, ElementFinder, ExpectedConditions, protractor } from 'protractor';
 
 export class Translate {
+    private readonly source: ElementFinder;
+    private readonly translation: ElementFinder;
+    private readonly openSource: ElementFinder;
+    private readonly targetSource: ElementFinder;
+    private readonly slSearch: ElementFinder;
+    private readonly tlSearch: ElementFinder;
+
+    constructor() {
```

Comment:
same about constructor (here it cost 9 extra lines in file which have ~50 lines :) )